app/markets/ActivityTable.tsx

```diff
@@ -0,0 +1,88 @@
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from "@/app/components/ui/Table";
+import { formatDateTime, shortenAddress } from "@/utils";
+import { Tag } from "swapr-ui";
+
+const activities = [
+  {
+    user: "0x8a05649a4fa186d032376f6d34f6bf40ca0168ae",
+    action: "Yes",
+    shares: "1.24",
+    date: 1718999911,
+  },
+  {
+    user: "0x1b3bbecdcb76594e1dc5f7550263cd5ea4a6ab6d",
+    action: "No",
+    shares: "2.3",
+    date: 1718999111,
+  },
+  {
+    user: "0x7532a3ce4d80cc3fb71d69a5329f76f9f375aa27",
+    action: "Yes",
+    shares: "32.2",
+    date: 1718912111,
+  },
+  {
+    user: "0x2fe3e7422f35ecbb9dca542279fb2ee172faab32",
+    action: "Yes",
+    shares: 23.1,
+    date: 1718903353,
+  },
+  {
+    user: "0x269e4c1cb7d8586a99baca522db8c2ef72bbf3ac",
+    action: "No",
+    shares: 12.2,
+    date: 1718901111,
+  },
+  {
+    user: "0xf9d7aeaa523906f59070eee225f16c7893e2b262",
+    action: "Yes",
+    shares: 0.42,
+    date: 1718801111,
+  },
+];
+
+export const ActivityTable = () => {
+  return (
+    <Table>
+      <TableHeader>
+        <TableRow>
+          <TableHead className="text-text-low-em">Users</TableHead>
+          <TableHead className="text-text-low-em">Action</TableHead>
+          <TableHead className="text-text-low-em">Shares</TableHead>
+          <TableHead className="text-text-low-em text-right">Date</TableHead>
+        </TableRow>
+      </TableHeader>
+      <TableBody className="font-semibold">
```

Comment:
Text can be bigger, right?

Reply:
it can, but I actually like this size, but I can increase it a bit to try out.

Reply:
Added text-base (which increases font in 2px)
<img width="593" alt="image" src="https://github.com/SwaprHQ/presagio/assets/5664434/0cd5d555-6e04-4068-8363-2f25f99080b2">

Reply:
I was suggesting 6 for the first one because we tend to memorize the first 4 digits of a certain (mostly personal) address. Last ones can be 3 to 4. We don't need to change it now, I just think we can improve and the box has space for it.